### ISSUE TITLE
Feature Request - Skip to next new comment

### ISSUE THREAD

**Issue by baudtack** [neagle/gokibitz#47]
It will often take days for people to comment on games. It would be nice if there was a way to skip to the next new comment you haven't seen. Right now I start from the beginning of a game and flip through them one by one till i find a name i didn't see before.

**Comment by neagle** [comment]
Yeah, I want this feature, too. It's in progress, and I hope to get it done sometime this week or weekend.

neagle/gokibitz/issues/40

**Comment by neagle** [comment]
Actually, let me reopen this. I haven't dealt with trying to mark which comment has actually been _seen_. I'll have to think about this, as figuring out how to do it in a straightforward way isn't easy.
